core/src/main/scala/org/apache/spark/util/collection/CompactBuffer.scala

```diff
@@ -112,8 +112,6 @@ private[spark] class CompactBuffer[T: ClassTag] extends Seq[T] with Serializable
 
   override def length: Int = curSize
 
-  override def size: Int = curSize
```

Comment:
This is not overrideable in 2.13, and in any event, just returns `.length` already in 2.12 anyway.